integration-tests/gradle/src/test/resources/conditional-dependencies/gradle.properties

```diff
@@ -0,0 +1,4 @@
+quarkusPlatformArtifactId=quarkus-bom
+quarkusPlatformGroupId=io.quarkus
+quarkusPlatformVersion=999-SNAPSHOT
```

Comment:
999-SNAPSHOT won't be available during the release process. The version should be added in the test initialization like in other gradle tests (or by resources filtering, if that's an option).

Reply:
Oh right, this is the case, I think it was for local tests